src/tasks/component-installers/devfile-workspace-operator-installer.ts

```diff
@@ -273,6 +282,38 @@ export class DevWorkspaceTasks {
           task.title = await `${task.title}...OK`
         }
       },
+      {
+        title: `Delete the Custom Resource of type ${this.cheManagerCRDName}`,
+        task: async (_ctx: any, task: any) => {
+          await this.kubeHelper.deleteCustomResource(this.devworkspaceCheNamespace, this.cheManagerApiGroupName, this.cheManagerApiVersionName, 'chemanagers')
+
+          // wait 10 seconds
+          await cli.wait(10000)
```

Comment:
It is better to poll every second like here
https://github.com/che-incubator/chectl/blob/main/src/tasks/installers/operator.ts#L521-L527